akka-stream-tck/src/test/scala/akka/stream/tck/ActorPublisherTest.scala

```diff
@@ -40,6 +40,8 @@ object ActorPublisherTest {
 
 class ActorPublisherTest extends AkkaPublisherVerification[Int] {
 
+  override lazy val system = createActorSystem()
```

Comment:
this will create another ActorSystem on top of the one created by AkkaPublisherVerification’s secondary constructor